airflow/sensors/external_task.py

```diff
@@ -102,6 +102,8 @@ class ExternalTaskSensor(BaseSensorOperator):
         If ``None`` (default value) the sensor waits for the DAG. Either
         external_task_id or external_task_ids can be passed to
         ExternalTaskSensor, but not both.
+    :param external_task_group_id: The task_group_id that contains the task you want to
+        wait for.
```

Comment:
Many operator classes add `(templated)` to templated fields and we should do that here as well.

Reply:
Went ahead and did this. Let me know if this is what you meant.